catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java

```diff
@@ -3125,6 +3130,15 @@ private String buildIngestLog(CreateRequest createReq) {
         }
     }
 
+    private String getAttributeStringValue(Metacard mcard, String attribute) {
+        Attribute attr = mcard.getAttribute(attribute);
+        if (attr != null && attr.getValue() != null) {
+            return attr.getValue()
+                    .toString();
+        }
+        return null;
```

Comment:
null seems risky since before it always had a String I doubt anything downstream is checking if this value is null.